app/routes/show-project.js

```diff
@@ -1,15 +1,25 @@
 import Route from '@ember/routing/route';
 import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
 
 export default class ShowProjectRoute extends Route {
+  @service
+  currentUser
+
   async model({ id }) {
     const project = await this.store.findRecord('project', id, {
       reload: true,
-      include: 'actions,milestones,dispositions',
+      include: 'actions,milestones,dispositions,users',
```

Comment:
At a glance, this doesn't seem necessary. This would be needed if code somewhere called `project.users`

Reply:
Cool, will remove

Reply:
hmm, that's interesting the test failed. I'll have to think/investigate more why. 
At a glance still seems like `project.users` is critical to the comparison `user.projects.map(proj => proj.dcp_name);`